spec/models/agents/evernote_agent_spec.rb

```diff
@@ -0,0 +1,297 @@
+require 'spec_helper'
+
```

Comment:
I think we should get some specs on the nested classes, but no strong preference on adding a gem vs. just adding specs.